evm_arithmetization/src/cpu/kernel/asm/mpt/storage/storage_write.asm

```diff
@@ -9,11 +9,26 @@ global sys_sstore:
     %stack (kexit_info, slot, value) -> (slot, kexit_info, slot, value)
     %sload_current
```

Comment:
Out of curiosity, the same issue doesn't happen with `SSTORE` operations triggering an OOG?

Reply:
Unlike with SLOAD, there's no way of knowing the gas cost of SSTORE without querying the state trie. So this isn't an issue for SSTORE.